### PR TITLE
feat(windows): implement Clear Logs, closes #3224

### LINKED ISSUE #3224: Implement "Clear logs" button
Need to implement the "Clear Logs" function

```
←[2m2024-01-14T15:11:19.676296Z←[0m ←[34mDEBUG←[0m ←[2mfirezone_windows_client::client::gui←[0m←[2m:←[0m started log counting
←[2m2024-01-14T15:11:22.004103Z←[0m ←[31mERROR←[0m ←[2mtracing_panic←[0m←[2m:←[0m A panic occurred ←[3mpanic.payload←[0m←[2m=←[0m"not yet implemented" ←[3mpanic.location←[0m←[2m=←[0m"windows-client\\src-tauri\\src\\client\\logging.rs:71:5"
←[2m2024-01-14T15:11:27.832726Z←[0m ←[34mDEBUG←[0m ←[2mfirezone_windows_client::client::gui←[0m←[2m:←[0m cancelled log counting
←[2m2024-01-14T15:11:29.220279Z←[0m ←[34mDEBUG←[0m ←[2mfirezone_windows_client::client::gui←[0m←[2m:←[0m started log counting
```

### PR DESCRIPTION
**Pull request by ReactorScram**

### CODE REVIEW

File: rust/windows-client/src-tauri/src/client/logging.rs [hunk 1]
```diff
@@ -67,8 +72,20 @@ pub(crate) async fn export_logs(managed: tauri::State<'_, Managed>) -> StdResult
         .map_err(|e| e.to_string())
 }
 
-pub(crate) async fn clear_logs_inner() -> Result<()> {
-    todo!()
+/// Delete all files in the logs directory.
+///
+/// This includes the current log file, so we won't write any more logs to disk
+/// until the file rolls over or the app restarts.
```
Comment (jamilbk) [inline rust/windows-client/src-tauri/src/client/logging.rs:78]: Hmm interesting side effect. I suspect we'll want to handle this edge case properly if it's not a ton more work. Would it be possible to just call `callbacks.roll_log_file` here similar to the Linux client?

https://github.com/firezone/firezone/blob/main/rust/linux-client/src/main.rs#L40

**Review by jamilbk — CHANGES_REQUESTED** [review]
LGTM other than we probably need to handle the rollover edge case. With our luck users will probably hit in on day 1 🤣

File: rust/windows-client/src-tauri/src/client/logging.rs (hunk 1 above)
Comment (ReactorScram) [reply rust/windows-client/src-tauri/src/client/logging.rs:78]: I added this as a task on #3211, it's hard to get the logging handle in the right place until that's closed.

Comment (jamilbk) [reply rust/windows-client/src-tauri/src/client/logging.rs:78]: sgtm

**Review by jamilbk — APPROVED** [review]

**Pull request merged**